recipes/transanno/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.4.5" %}
+{% set sha256 = "f800ef6d9eff31aad6e00bbe963fd0c57a9ba9036c676a972ceefd9d1cdfbdc9" %}
+{% set name = "transanno" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x.x") }}
+
+source:
+  url: https://github.com/informationsea/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - rust >=1.43
+    - {{ compiler('rust') }}
+
+test:
+  commands:
+    - transanno --help
+
+about:
+  home: https://github.com/informationsea/{{ name }}
+  license: GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE
+  summary: "accurate VCF/GFF3/GTF LiftOver tool for new genome assemblies"
+
+extra:
+  additional-platforms:
+    - osx-arm64
```

Comment:
```suggestion
    - osx-arm64
    - linux-aarch64
```

Reply:
Thanks, I had problems with that before but it might have been an intermittent failure.

Reply:
and now it builds for `linux-aarch64`! all tests passed!